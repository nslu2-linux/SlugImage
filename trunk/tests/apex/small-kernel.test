#!/bin/sh
image=$(tempfile)
../slugimage -d -p -o $image -L files/128KiB-16 -k files/896KiB-16 -r Flashdisk:files/6528KiB
hexdump -C $image > $1
rm -rf $image
