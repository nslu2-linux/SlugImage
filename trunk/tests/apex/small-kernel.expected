00000000  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
0003ffb0  00 00 00 00 00 00 00 00  00 00 73 45 72 43 6f 4d  |..........sErCoM|
0003ffc0  6d 00 01 00 00 04 70 31  95 58 10 00 00 00 00 00  |m.....p1.X......|
0003ffd0  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
0003ffe0  00 00 00 00 00 00 00 00  01 00 00 00 00 00 00 00  |................|
0003fff0  03 00 00 23 63 00 00 00  20 73 45 72 43 6f 4d 6d  |...#c... sErCoMm|
00040000  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
00060000  00 01 ff f0 00 00 00 00  00 00 00 00 00 00 00 00  |................|
00060010  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
00080000  00 0f ff f0 00 00 00 00  00 00 00 00 00 00 00 00  |................|
00080010  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
00160000  00 02 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
00160010  ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff ff  |................|
*
00180000  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
007e0000  52 65 64 42 6f 6f 74 00  00 00 00 00 00 00 00 00  |RedBoot.........|
007e0010  50 00 00 00 00 00 00 00  00 04 00 00 00 00 00 00  |P...............|
007e0020  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
007e0100  53 79 73 43 6f 6e 66 00  00 00 00 00 00 00 00 00  |SysConf.........|
007e0110  50 04 00 00 00 00 00 00  00 02 00 00 00 00 00 00  |P...............|
007e0120  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
007e0200  4c 6f 61 64 65 72 00 00  00 00 00 00 00 00 00 00  |Loader..........|
007e0210  50 06 00 00 00 00 00 00  00 02 00 00 00 00 00 00  |P...............|
007e0220  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
007e02e0  00 00 00 00 00 00 00 00  00 00 00 00 73 6b 69 70  |............skip|
007e02f0  00 00 00 00 00 00 00 10  00 00 00 00 00 00 00 00  |................|
007e0300  4b 65 72 6e 65 6c 00 00  00 00 00 00 00 00 00 00  |Kernel..........|
007e0310  50 08 00 00 00 00 00 00  00 10 00 00 00 00 00 00  |P...............|
007e0320  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
007e03e0  73 6b 69 70 00 00 00 00  00 00 00 10 73 6b 69 70  |skip........skip|
007e03f0  00 0e 00 00 00 00 00 10  00 00 00 00 00 00 00 00  |................|
007e0400  46 6c 61 73 68 64 69 73  6b 00 00 00 00 00 00 00  |Flashdisk.......|
007e0410  50 18 00 00 00 00 00 00  00 66 00 00 00 00 00 00  |P........f......|
007e0420  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
007e0500  46 49 53 20 64 69 72 65  63 74 6f 72 79 00 00 00  |FIS directory...|
007e0510  50 7e 00 00 00 00 00 00  00 02 00 00 00 00 00 00  |P~..............|
007e0520  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
007e0600  ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff ff  |................|
*
007ffff0  00 01 00 00 23 63 90 f7  65 52 63 4f 6d 4d 00 b4  |....#c..eRcOmM..|
00800000
